src/openapi/streaming/subscription.js

```diff
@@ -287,15 +290,15 @@ function onSubscribeSuccess(referenceId, result) {
  * Called when a subscribe errors
  * @param response
  */
-function onSubscribeError(referenceId, response) {
+function onSubscribeError(referenceId, body) {

```

Comment:
I don't think this is body - its an object
```
{
    response: {},
    status: 404,
}
```
I know its ugly doing response.response - thats historical. I think I wanted 
```
{
  body,
  status,
}
```
being the response